deploy/cluster_role.yaml

```diff
@@ -27,25 +18,21 @@ rules:
   - update
   - watch
 - apiGroups:
-  - apps
+  - ""
   resources:
-  - deployments
-  - deployments/finalizers
-  - daemonsets
-  - replicasets
-  - statefulsets
+  - persistentvolumeclaims
```

Comment:
why do we need PVC?

Reply:
Thank you for your review @dofinn 
PVC access is not required. I will test after removing it.